pages.it/common/git-config.md

```diff
@@ -0,0 +1,29 @@
+# git config
+
+> Configura le impostazioni di uno o piu repository git.
+> Tali voci di configurazione possono essere sia locali (per il repository corrente) che globali (per l'utente corrente).
+> Maggiori informazioni: <https://git-scm.com/docs/git-config>.
+
+- Elenca solo le voci di configurazione locali (salvate in `.git/config` nel repository corrente):
```

Comment:
```suggestion
- Elenca solo le opzioni di configurazione locali (salvate in `.git/config` nel repository corrente):
```
I'd suggest changing "voci" to "opzioni" here and in the rest of the page. It's a more correct term to use.